keps/sig-node/20190131-runas-groupid.md

```diff
@@ -14,21 +13,266 @@ approvers:
   - "@derekwaynecarr"
 editor: TBD
 creation-date: 2017-06-21
-last-updated: 2017-09-14
+last-updated: 2019-02-14
 status: implementable
 ---
 
 # RunAsGroup support in PodSpec and PodSecurityPolicy
 
-https://github.com/kubernetes/community/blob/master/contributors/design-proposals/auth/runas-groupid.md
-
 ## Table of Contents
 
+* [Abstract](#abstract)
+
+* [Motivation](#motivation)
+
+    * [What is the significance of Primary Group Id ?](#what-is-the-significance-of-primary-id)
+
+* [Goals](#goals)
+
+* [Use Cases](#use-cases)
+
+    * [Use Case 1](#use-case-1)
+    * [Use Case 2](#use-case-2)
+
+* [Design](#design)
+
+    * [Model](#model)
+
+        * [SecurityContext](#security-context)
+
+        * [PodSecurityContext](#pod-security-context)
+
+        * [PodSecurityPolicy](#pod-security-policy)
+
+# [Behavior](#behavior)
+
+# [Summary of Changes needed](#summary-of-changes-needed)
```

Comment:
Fix ToC - this and the above should use `*` rather than `#`. Also you can remove the line breaks between each item.

Reply:
done